src/core/client/admin/routeConfig.tsx

```diff
@@ -45,17 +48,49 @@ export default makeRouteConfig(
         <Route path="moderate" {...ModerateRoute.routeConfig}>
           <Redirect from="/" to="/admin/moderate/reported" />
           <Route path="reported" {...ReportedQueueRoute.routeConfig} />
-          <Route path="reported/:storyID" {...ReportedQueueRoute.routeConfig} />
+          <Route
```

Comment:
note new URLs for moderation queues. Need to be able to distinguish between site ID and story ID.